tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -33,7 +33,19 @@
 """Schema separately parses translation to account for multiple valid formats
 github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md
 #defining-translations"""
-_TRANSLATION_SCHEMA = syaml.Str() | syaml.Any()
+_TRANSLATION_SCHEMA = syaml.Str() | syaml.MapPattern(
+    syaml.Str(),
+    syaml.Str() | syaml.Map({'present_value': syaml.Str(),
+                             syaml.Optional('states'): syaml.MapPattern(
+                                 syaml.Str(), syaml.Str()),
```

Comment:
this is always CAP[_LTRS]

Reply:
are you referring to states?

we can use an enum: https://hitchdev.com/strictyaml/using/alpha/scalar/enum/
but lets keep it lower case for simplicity

Reply:
Enums are always CAPS, including in our docs.